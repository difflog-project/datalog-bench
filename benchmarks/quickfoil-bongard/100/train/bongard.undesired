8968187
1258535
8703926
4406136
6004512
1630932
93610
1179878
4800862
3239242
7723464
3339008
862504
8959595
3251988
9105389
3118779
665685
2458329
2615567
1192128
8896092
7878219
3505492
6323252
6650892
8648602
3720272
3203745
6982868
811715
5667526
2728595
7886709
209676
49059
9048838
5324244
5857164
973536
1784092
6483510
2294606
8125750
4401603
7751971
8832957
3238099
7025089
4245419
386590
5442317
8487546
1468233
8200788
3855496
2334691
607043
5332405
4821734
4289927
4547256
4735199
159212
197571
1481859
9142888
6987659
6701808
5802639
6550125
5103831
4955372
18142
960354
2101624
1855327
5160827
6039952
90606
2130806
7482418
1329337
4050652
3336522
976861
2922936
3861237
3948690
3924901
