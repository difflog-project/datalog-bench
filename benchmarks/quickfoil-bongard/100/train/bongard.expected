1267895
8004111
2085117
8457418
5897097
512354
8821271
7983408
1700286
1809102
1146711
8419565
5282290
8877931
2311965
5217506
1242591
4172431
2660031
921011
9323793
2583249
9536288
1473389
8596762
6492539
275706
9298251
8909734
9752818
9103852
9542428
3865291
3789353
8574751
5860794
1036569
5270429
8581693
6418129
9656557
5461615
3897432
8424772
5335693
2354497
2300099
4756709
5268371
1753056
4071836
8834957
2735850
6989005
9701941
7515025
9153765
425854
6691079
6775388
7697919
625421
509422
6803095
3735825
7505483
9952664
861754
2129111
2005782
3032913
890796
6860881
6329506
1983048
2929023
1699529
2484582
4407055
9912
8222819
6346444
3305912
7358781
6306437
6263080
8936235
4824714
7852250
3415277
